docs/plugin_advantages.md

```diff
@@ -0,0 +1,52 @@
+# Advantages of using this Plugin
+
+## Handles Windows Init
+One of the core features of the Unity Plugin for EOS, is that it handles
+doing the init code in such a way that the Overlay will work correctly in 
+standalone builds on windows. Due to how the EOS SDK does it's graphics initialization 
+for the overlay, the code that call's into the EOS SDK for init needs to happen _before_
+the Unity finishes doing it's graphics init code. The only place to do this, is in native
+code, in a specially named DLL that gets called before the engine finishes running. 
+If one is using this Plugin, one doesn't have to worry about any of that, as it's handled 
+automatically.
+
+
+## Handles the Unity Editor
+Need to unload and reload the EOS SDK dynamic libraries to ensure new config works
```

Comment:
Is this a directive or an explanation of functionality? Supposed to be a complete sentence?

Reply:
That is a line that needs to be deleted: thank you!